### ISSUE TITLE
Is osicat-posix in the doc ?

### ISSUE THREAD

**Issue by vindarel** [osicat/osicat#20]
Hi,
Osicat was my go-to package to look for how to get a file size and attributes. Ufortunately I found nothing related in the doc, but it appears there is `osicat-posix:stat`.

It should either be in the doc, either more visible !

**Comment by luismbo** [comment]
Agreed. osicat-posix should be documented. It'd be super cool if we could (automatically) adapt the POSIX documentation, too.
